includes/admin.php

```diff
@@ -13,6 +13,13 @@ function drift_options_page()
     wp_cache_flush();
   }
 
+  // Check to see if Drift identify is checked
```

Comment:
this code (and all code in https://github.com/Driftt/drift-wordpress/pull/13) was pushed to the SVN repo a year ago and is thus stable; adding it back here to not cause any unexpected regressions